<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu2</class>
 <widget class="QMainWindow" name="Menu2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>10</y>
      <width>1531</width>
      <height>821</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>281</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_2{
background-color: rgb(107, 107, 107);
background:rgba(0/0/0/0.3);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="editarMedico">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Ir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Editar Médico</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Menú</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>380</y>
       <width>281</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_5{
background-color: rgb(107, 107, 107);
background:rgba(0/0/0/0.3);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="editarCita">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Ir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Editar Cita</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_8">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>600</y>
       <width>281</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_8{
background-color: rgb(107, 107, 107);
background:rgba(0/0/0/0.3);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="regresar">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Ir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>160</y>
       <width>281</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_3{
background-color: rgb(107, 107, 107);
background:rgba(0/0/0/0.3);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Editar Paciente</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editarPaciente">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Ir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>380</y>
       <width>281</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_6{
background-color: rgb(107, 107, 107);
background:rgba(0/0/0/0.3);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Eliminar Consulta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eliminarConsulta">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Ir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_9">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>160</y>
       <width>281</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_9{
background-color: rgb(107, 107, 107);
background:rgba(0/0/0/0.3);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Editar Consulta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editarConsulta">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Ir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_7">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>380</y>
       <width>281</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_7{
background-color: rgb(107, 107, 107);
background:rgba(0/0/0/0.3);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Eliminar Cita</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eliminarCita">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Ir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>Menú</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRegresar"/>
   </widget>
   <addaction name="menuMen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegresar">
   <property name="text">
    <string>Cerrar Sesion</string>
   </property>
  </action>
  <action name="actionCerrar_Sesion">
   <property name="text">
    <string>Cerrar Sesion</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Imagenes/Imagen.qrc"/>
 </resources>
 <connections/>
</ui>
